new_builder_evaluation/dynalab/dynalab.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) MLCommons and its affiliates.
```

Comment:
Should we think about a cleaner way to do this? How can we actually get the input information for all of the tasks so this can be done automatically?